modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/ContextMenuContent.java

```diff
@@ -810,6 +819,14 @@ Menu getOpenSubMenu() {
         return openSubmenu;
     }
 
+    boolean isUpArrowVisible() {
```

Comment:
If these two new methods are only added for testing (which it looks like they are), please add a comment to that effect.